src/reflect/scala/reflect/internal/Constants.scala

```diff
@@ -60,6 +63,8 @@ trait Constants extends api.Constants {
     def isFloatRange: Boolean = ByteTag <= tag && tag <= FloatTag
     def isNumeric: Boolean    = ByteTag <= tag && tag <= DoubleTag
     def isNonUnitAnyVal       = BooleanTag <= tag && tag <= DoubleTag
+    def isSymbol: Boolean     = tag == SSymbolTag
```

Comment:
I don't think we should add support for Symbol constants. This will break older compilers, right?